AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -856,9 +867,6 @@ - (void)transitionLayoutWithSizeRange:(ASSizeRange)constrainedSize
     ASDN::MutexLocker l(__instanceLock__);
     ASDisplayNodeAssert(ASHierarchyStateIncludesLayoutPending(_hierarchyState) == NO, @"Can't start a transition when one of the supernodes is performing one.");
   }
-
-  // Invalidate the current layout to be able to measure a new layout based onthe given constrained size
-  [self setNeedsLayout];
```

Comment:
@maicki I thin this line is still needed in the case this method is called directly without going through the one above?